yarn.lock

```diff
@@ -3110,7 +3110,7 @@ file-entry-cache@^2.0.0:
     flat-cache "^1.2.1"
     object-assign "^4.0.1"
 
-file-loader@^1.1.10:
```

Comment:
The lock file is being changed but the package.json is not updated. I would run `yarn upgrade-dependencies --latest` to update also the package.json

Reply:
I'm just going to revert this change as there are quite a few other changes after I run `yarn upgrade` and I think it would be better to keep them out of this PR

Reply:
👍 I updated the command to `upgrade-interactive`